src/components/BaseInfo/BaseInfoStyles.js

```diff
@@ -0,0 +1,22 @@
+import styled from '@emotion/styled';
+
+export const Title = styled.h2`
+  color: #444;
+  margin: 0;
+  margin-bottom: 16px;
+  font-weight: 700;
+  font-size: 18.72px;
+`;
+
+export const Image = styled.img`
+  width: 100%;
+  overflow: hidden;
+  border-radius: 5px;
```

Comment:
better use here border-radius: inherit, because in future if you will want to change border-radius of ImageContainer, you will have to change only there.